manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LDraw.AIOI
+PackageVersion: 9.5.0.0
+PackageLocale: en-US
+Publisher: LDraw

```

```suggestion
Publisher: LDraw.org
```
Must match Control Panel / Apps&Features